__tests__/app.test.js

```diff
@@ -31,31 +31,64 @@ describe("GET /api/topics", () => {
   });
 });
 
-// describe("GET /api/articles", () => {
-//   test("200: returns an object with an array with correctly formatted objects", () => {
-//     return request(app)
-//       .get("/api/articles")
-//       .expect(200)
-//       .then(({ body }) => {
-//         const { articles } = body;
-//         expect(articles).toBeInstanceOf(Array);
-//         expect(articles).toHaveLength(12);
-//         articles.forEach((article) => {
-//           expect(article).toEqual(
-//             expect.objectContaining({
-//               author: expect.any(String),
-//               title: expect.any(String),
-//               article_id: expect.any(Number),
-//               topic: expect.any(String),
-//               created_at: expect.any(String),
-//               vote: expect.any(Number),
-//               comment_count: expect.any(Number),
-//             })
-//           );
-//         });
-//       });
-//   });
-// });
+describe("GET /api/articles", () => {
+  test("200: returns an object with an array with correctly formatted objects sorted by article ID in ascending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(5);
+        // expect(articles).toBeSortedBy("article_id") CANT GET THIS TO WORK. TRIED TO CHANGE JEST SCRIPT BUT MESSED OTHER THINGS UP;
```

Comment:
you could check that the first article to come back has a specific date for the time being?